src/current/v24.3/configure-replication-zones.md

```diff
@@ -59,19 +65,257 @@ In addition, CockroachDB stores internal [**system data**]({% link {{ page.versi
 
 Level | Description
 ------|------------
-------|------------
 Cluster | The `default` replication zone mentioned above also applies to all system ranges not constrained by a more specific replication zone.
 System Range | CockroachDB comes with pre-configured replication zones for important system ranges, such as the "meta" and "liveness" ranges. If necessary, you can add replication zones for the "timeseries" range and other system ranges as well. Editing replication zones for system ranges may override settings from `default`. See [Create a Replication Zone for a System Range](#create-a-replication-zone-for-a-system-range) for more details.<br><br>CockroachDB also comes with pre-configured replication zones for the internal `system` database and the `system.jobs` table, which stores metadata about long-running jobs such as schema changes and backups.
 
 ### Level priorities
 
 When replicating data, whether table or system, CockroachDB always uses the most granular replication zone available. For example, for a piece of user data:
 
-1. If there's a replication zone for the row, CockroachDB uses it.
-1. If there's no applicable row replication zone and the row is from a secondary index, CockroachDB uses the secondary index replication zone.
-1. If the row isn't from a secondary index or there is no applicable secondary index replication zone, CockroachDB uses the table replication zone.
-1. If there's no applicable table replication zone, CockroachDB uses the database replication zone.
-1. If there's no applicable database replication zone, CockroachDB uses the `default` cluster-wide replication zone.
+1. If there's a replication zone [for the row](#create-a-replication-zone-for-a-partition) (a.k.a. [partition]({% link {{ page.version.version }}/partitioning.md %})), CockroachDB uses it.
+1. If there's no applicable row replication zone and the row is from a secondary index, CockroachDB uses the [secondary index replication zone](#create-a-replication-zone-for-a-secondary-index).
```

Comment:
i wonder why the docs before also just specified secondary index -- can we generalize "secondary indexes" to just "indexes" (or was this specification intentional)? 

primary indexes can be configured as such:

```
demo@127.0.0.1:26257/demoapp/movr> CREATE TABLE foo(i INT PRIMARY 
                                -> KEY, j INT) PARTITION BY LIST  
                                -> (i) (                          
                                ->   PARTITION p VALUES IN (1, 5) 
                                -> );                             

demo@127.0.0.1:26257/demoapp/movr> ALTER INDEX foo@foo_pkey       
                                -> CONFIGURE ZONE USING           
                                -> num_replicas = 11; 
```

Reply:
thanks for catching this, idk why it says "secondary index" and is so specific about it, i don't think there is a serious intent behind that, i think it's just how the person writing the docs did it at one time. looking at git it's a change from v1.2 so i think it's just historical accident 

went through here and a few other places in this doc that said "secondary index" and changed to just say "index"